commit 95d382819500ac9b8cad358e275bcfdffbd6dd96
Author: root <root@minos47-2.zcu.cz>
Date:   Thu Jul 16 03:30:38 2015 +0200

    meta-defaultcfg.patch
    preservefqdn, stats, ratelimit

diff --git a/debian/rsyslog.conf b/debian/rsyslog.conf
index 0e33f48..803024c 100644
--- a/debian/rsyslog.conf
+++ b/debian/rsyslog.conf
@@ -3,14 +3,24 @@
 #			For more information see
 #			/usr/share/doc/rsyslog-doc/html/rsyslog_conf.html
 
+$PreserveFQDN on
 
 #################
 #### MODULES ####
 #################
 
-$ModLoad imuxsock # provides support for local system logging
-$ModLoad imklog   # provides kernel logging support
-#$ModLoad immark  # provides --MARK-- message capability
+module(load="imuxsock") # provides support for local system logging
+module(load="imklog")   # provides kernel logging support
+module(load="immark")  # provides --MARK-- message capability
+
+$ModLoad impstats
+$PStatInterval 600
+$PStatSeverity 7
+$PStatJSON on
+ 
+$IMUXSockRateLimitInterval 3600
+$IMUXSockRateLimitBurst 100000
+$RepeatedMsgReduction on
 
 # provides UDP syslog reception
 #$ModLoad imudp
