commit 6f39ceaaa61cfce6727b218214af8323d58cd357
Author: root <root@minos47-2.zcu.cz>
Date:   Thu Jul 16 03:19:50 2015 +0200

    meta-buildcontrols.patch
    changelog, control author, notest(fails), noquilt

diff --git a/debian/changelog b/debian/changelog
index 0b01623..66ae525 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,10 @@
+rsyslog (8.4.2+deb8u1.rb30) jessie; urgency=medium
+
+  * Non maintainer upload
+  * metacentrum.cz specific changes
+
+ -- Radoslav Bodo <bodik@cesnet.cz>  Mon, 23 Jun 2014 13:44:55 +0200
+
 rsyslog (8.4.2-1+deb8u1) jessie; urgency=medium
 
   * Disable transactions in ompgsql as they were not working properly.
diff --git a/debian/control b/debian/control
index 1eb7f6c..eee2999 100644
--- a/debian/control
+++ b/debian/control
@@ -1,7 +1,7 @@
 Source: rsyslog
 Section: admin
 Priority: important
-Maintainer: Michael Biebl <biebl@debian.org>
+Maintainer: Radoslav Bodo <bodik@cesnet.cz>
 Build-Depends: debhelper (>= 8),
                dpkg-dev (>= 1.16.1),
                dh-autoreconf,
diff --git a/debian/source/format b/debian/source/format
index 163aaf8..89ae9db 100644
--- a/debian/source/format
+++ b/debian/source/format
@@ -1 +1 @@
-3.0 (quilt)
+3.0 (native)
